lib/internal/test_runner/utils.js

```diff
@@ -27,6 +33,13 @@ const {
 } = require('internal/errors');
 const { compose } = require('stream');
 
+const coverageColors = {
+  '__proto__': null,
+  'high': green,
+  'medium': '\u001b[33m',
```

Comment:
please add `yellow` to https://github.com/nodejs/node/blob/ab8942848b9967e6e0289ff7f1a8b12a349b7a28/lib/internal/util/colors.js